poker_ai/clustering/runner.py

```diff
@@ -357,7 +356,7 @@ def process_flop_potential_aware_distributions(
         available_cards: np.ndarray = self.get_available_cards(
             cards=self._cards, unavailable_cards=public
         )
-        potential_aware_distribution_flop = np.zeros_like(self._turn_centroids)
+        potential_aware_distribution_flop = [0] * len(self._turn_centroids)
```

Comment:
Returning a numpy array from this method when this object was also array created a 3d matrix that the sklearn clustering couldn't support.